Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -484,6 +525,14 @@ class SettingsTableViewController: UITableViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        view.backgroundColor = UIColor.clearColor()
```

Comment:
I don't know what this change was for but I notice that now the background of the SettingsTableViewController is black when you pull down to refresh. Do we need this?